CHANGES.rst

```diff
@@ -12,6 +12,24 @@
 Changelog
 =========
 
+5.1.5 (unreleased)
+------------------
+
+Breaking changes:
+
+- *add item here*
+
+New features:
+
+- Add ``load_async`` and ``load_defer`` attributes to resource registries bundle settings.
+  When set, ``<script>`` tags are rendered with ``async="async"`` resp. ``defer="defer"`` attributes.
+  [thet]
```

Comment:
this should go to a file within ``news`` folder. See the comment on https://raw.githubusercontent.com/plone/Products.CMFPlone/master/CHANGES.rst for details.